lib/utils/cli-options.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const _ = require('lodash');
+const chalk = require('chalk');
+const opts = require("nomnom")
+    .option('development', {
+        abbr: 'dev',
+        flag: true,
+        help: 'Run with process.env.COINS_ENV set to \'development\''
+    })
+    .option('release', {
+        abbr: 'rel',
+        flag: true,
+        help: 'Run with process.env.COINS_ENV set to \'release\''
+    })
+    .option('production', {
+        abbr: 'prd',
+        flag: true,
+        help: 'Run with process.env.COINS_ENV set to \'production\''
+    })
+    .parse();
+
+_.each(opts, (val, opt) => {
+    if (_.contains(['development', 'release', 'production'], opt) && val) {
+        console.log(chalk.blue(`Running with COINS_ENV: ${opt}`));
```

Comment:
How about moving this line to after line 27? That way we'll get a print out of what environment we're in all the time.